### PR TITLE
Improved docs for mmv1 options

### PR DESCRIPTION
**Pull request by melinath**
I think this is worth documenting since I've had to look it up a few times for myself or other people.

```release-note:none

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,15 +121,22 @@ It's worth noting that Magic Modules will only generate new files when ran
...
-and using the repository for the `google-beta` provider.
+`-e`, `-v`, and `-f` let you select which project should be generated.
 
-```bash
-bundle exec compiler -a -v "beta" -e terraform -o "$GOPATH/src/github.com/hashicorp/terraform-provider-google-beta"
-```
+Target                      | compiler options
+----------------------------|-----------------
+ansible                     | -e ansible
+inspec                      | -e inspec
```
Comment (slevenick) [inline README.md]:
```suggestion
inspec                      | -e inspec -v "beta"
```

Inspec is expected to be compiled at beta

File: README.md [hunk 2]
```diff
@@ -121,15 +121,22 @@ It's worth noting that Magic Modules will only generate new files when ran
 locally. The "Magician"- the Magic Modules CI system- handles deletion of old
 files when creating PRs.
 
-#### Terraform's `google-beta` provider
+#### Compiler options
 
-Terraform is the only tool to handle Beta features right now; you can generate
-`google-beta` by running the following, substitution `"beta"` for the version
-and using the repository for the `google-beta` provider.
+`-e`, `-v`, and `-f` let you select which project should be generated.
```
Comment (slevenick) [inline README.md:126]: `-v` is a version flag rather than selecting a project I believe

**Review by slevenick — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (melinath) [reply README.md:126]: yes, that's correct; what I'm trying to convey here is that each output project (tpg vs tpgb vs inspec vs tgc) is uniquely identified by some combination of these three options. Maybe that's not the best way to explain it? Like, `-f` technically means "force using a non-default provider" but really from the end user perspective I think the important thing isn't what it means but rather what combination they need to get the right output for the project they're working on.

Internally to MM, the version flag indicates whether beta or ga things are being selected, but that's not a meaningful choice for every potential project you might generate.

Does that make sense? What do you think / do you have any suggestions for better phrasing?

File: README.md (hunk 1 above)
Comment (melinath) [reply README.md]: ah, good to know!

File: README.md (hunk 2 above)
Comment (slevenick) [reply README.md:126]: Yeah, I guess to me tpg and tpgb are the same project but different versions while`-f` definitely chooses a different project.

No big preference from me though, this seems fine

Comment (melinath) [reply README.md:126]: ok - I'll merge it as-is and if it causes problems we can always update again later :-)

**Pull request merged**